src/EnrollWindowsHelloController.js

```diff
@@ -110,25 +126,75 @@ function (Okta, FormController, FormType, webauthn, Footer, WindowsHelloErrorMes
 
       formChildren: function () {
         var result = [];
+
         if (!webauthn.isAvailable()) {
           result.push(
             FormType.View(
-              { View: WindowsHelloErrorMessageView },
-              { selector: '.o-form-error-container' }
+              { View: new WindowsHelloErrorMessageView(
+                { message: Okta.loc('enroll.windowsHello.error.notWindows', 'login') }) },
+              { selector: '.o-form-error-container'}
             )
           );
         }
+        else {
+          result.push(FormType.View({
```

Comment:
Just a note since I didn't catch this in my original review - is there a reason we're waiting on states for the subtitle instead of just rendering? I.e. I would expect this logic to be in the subtitle property, and whenever we change the enrollmentState just to re-render the entire form. 

Is the reason to do it this way because re-rendering with the spinner transitions poorly?

The main reason to do it the re-rendering way is because we are now copying subtitle elements and classes here - if we make a change to how the elements look in the underlying form view, we will need to remember to update it here.